adafruit_lsm9ds0.py

```diff
@@ -379,8 +372,9 @@ def _read_bytes(self, sensor_type, address, count, buf):
             device = self._xm_device
         with device as i2c:
             buf[0] = address & 0xFF
-            i2c.write(buf, end=1, stop=False)
+            i2c.write(buf, end=1)
             i2c.readinto(buf, end=count)
+        #print("read from %02x: %s" % (address, [hex(i) for i in buf[:count]]))
```

Comment:
Please remove `print` statement. Seems like it's left over from debugging, shouldn't be needed in the final release.